.github/workflows/changelog_verifier.yml

```diff
@@ -15,4 +15,4 @@ jobs:
 
       - uses: dangoslen/changelog-enforcer@v3
         with:
-          skipLabels: "autocut"
\ No newline at end of file
+          skipLabels: "autocut, skip-changelog"
```

Comment:
is it `"autocut, skip-changelog"` or `"autocut", "skip-changelog"`

Reply:
Seems to be the first. https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/changelog_verifier.yml#L18